lib/mayaUsd/ufe/UsdStageMap.cpp

```diff
@@ -83,6 +86,16 @@ inline Ufe::Path::Segments::size_type nbPathSegments(const Ufe::Path& path)
 #endif
 }
 
+inline Ufe::Path toPath(const std::string& pathString)
```

Comment:
I almost feel like we should put this in some util function, it has come up a few times in the past... I know I've been bitten a lot by it in unit test (but that is in Python, irrelevant here.)